testsuite/forge-cli/src/main.rs

```diff
@@ -417,7 +426,8 @@ fn single_test_suite(test_name: &str) -> Result<ForgeConfig<'static>> {
             })),
         "state_sync" => config
```

Comment:
any reason you need 30 validators here? you are not really testing validators? Also would it be better to have multiple full nodes, to catch issues more often? i.e. that's what I did for state-sync test here - https://github.com/aptos-labs/aptos-core/pull/3609

@sitalkedia what about changing default here to like 5, and then having specific tests make it larger if they actually need it?

Reply:
>what about changing default here to like 5, and then having specific tests make it larger if they actually need it?

Yes, that makes sense to me. Feel free to make that change or I will add that as a part of my subsequent test.